eureka-service-discovery/src/main/resources/application.yml

```diff
@@ -1,11 +1,22 @@
 server:
   port: 8761
 
+#Eureka properties
 eureka:
+  #Eureka setting for other service run outside the docker-container
   instance:
     hostname: localhost
   client:
     registerWithEureka: false
     fetchRegistry: false
     serviceUrl:
-      defaultZone: http://${eureka.instance.hostname}:${server.port}/eureka/
\ No newline at end of file
+      defaultZone: http://${eureka.instance.hostname}:${server.port}/eureka/
+
+
+#Eureka setting for other service run inside the docker-container
+---
+spring:
+  profiles: docker
+eureka:
+  instance:
+    hostname: eureka
```

Comment:
Is this working without this configurations?

Reply:
I can't check if other services in docker are working without this configurations. But I know other services in local machine are working without this configurations.

Reply:
I don't know how to check this